tests/RLS/TableManagerTest.php

```diff
@@ -534,6 +541,99 @@
     ]);
 })->with([true, false]);
 
+test('table owner sees all the records when forceRls is false while other users only see records scoped to them', function() {
+    CreateUserWithRLSPolicies::$forceRls = false;
+
+    // Drop all tables created in beforeEach
+    DB::statement("DROP TABLE authors, categories, posts, comments, reactions, articles;");
+
+    [$username, $password] = createPostgresUser('central_user');
+
+    config(['database.connections.central' => array_merge(
+        config('database.connections.pgsql'),
+        ['username' => $username, 'password' => $password]
+    )]);
+
+    DB::reconnect();
+
+    Schema::create('orders', function (Blueprint $table) {
+        $table->id();
+        $table->string('name');
+
+        $table->string('tenant_id')->comment('rls');
+        $table->foreign('tenant_id')->references('id')->on('tenants')->onUpdate('cascade')->onDelete('cascade');
+
+        $table->timestamps();
+    });
+
+    [$tenant1, $tenant2] = [Tenant::create(), Tenant::create()];
+
+    pest()->artisan('tenants:rls');
+
+    [$order1, $order2] = [
+        Order::create(['name' => 'order1', 'tenant_id' => $tenant1->getTenantKey()]),
+        Order::create(['name' => 'order2', 'tenant_id' => $tenant2->getTenantKey()]),
+    ];
+
+    // The table owner should see all the records
+    expect(Order::all())->toHaveCount(2);
```

Comment:
In this test, add a dataset for `$forceRls` so the difference is actually observed. Assuming `createPostgresUser()` creates a user without BYPASSRLS, the change should be simple.

Reply:
Added the dataset here https://github.com/archtechx/tenancy/pull/1293/commits/b7dda966a731531de4c671fab2cd9291253021af (also fixed the failing test in the commit)